File: tests/fixtures/multilang/sample.go
package sample

import "fmt"

// Add adds two ints.
func Add(a int, b int) int {
	if a > 0 {
		return helper.Scale(a, b)
	}
	return a + b
}

type Widget struct{}

// Render renders w.
func (w *Widget) Render() {
	w.Add(1, 2)
}

File: tests/fixtures/multilang/sample.rs
use std::fmt;

/// Adds two numbers.
fn add(a: i32, b: i32) -> i32 {
    if a > 0 {
        return helper::scale(a, b);
    }
    a + b
}

struct Widget;

impl Widget {
    // Renders the widget.
    fn render(&self) {
        self.add(1, 2);
    }
}
